src/MojiraBot.ts

```diff
@@ -24,6 +24,25 @@ export default class MojiraBot {
 	private static running = false;
 
 	public static async start(): Promise<void> {
+		const logConfig: log4js.Configuration = {
+			appenders: {
+				out: { type: 'stdout' },
+			},
+			categories: {
+				default: { appenders: [ 'out' ], level: BotConfig.debug ? 'debug' : 'warn' },
```

Comment:
I was unsure here whether the default level should be `info` or `warn`.
The only thing the `debug` flag seemed to do previously was printing a log message.

Reply:
Yeah, debug mode actually doesn't really do anything currently. This way it seems good though. `warn` is a good default level for non-debug mode in my opinion, as messages with default level might or might not be important (since level is not specified), so you should definitely be able to see them.